examples/tutorials/sst2_classification_non_distributed.py

```diff
@@ -0,0 +1,156 @@
+"""
+SST-2 Binary text classification with XLM-RoBERTa model
+=======================================================
+
+**Author**: `Parmeet Bhatia <parmeetbhatia@fb.com>`__
+
+"""
+
+######################################################################
+# Overview
+# --------
+#
+# This tutorial shows how to train a text classifier using pre-trained XLM-RoBERTa model.
+# TODO
+
+
+######################################################################
+# Common imports
+# --------------
+
+import torch
+import torch.nn as nn
+DEVICE = torch.DEVICE("cuda") if torch.cuda.is_available() else "cpu"
```

Comment:
Did you use some sort of find and replace?
I do not think this is what you intended.

Reply:
Thanks for catching (since I was testing on CPU, this code never executed ;) )